HelpSource/Classes/Slider2D.schelp

```diff
@@ -74,6 +74,11 @@ METHOD:: knobColor
 	argument::
 		A Color.
 
+METHOD:: thumbSize
+	The size of the handle.
```

Comment:
It could be worth mentioning that this is a diameter of handle (I may have thought radius).